.pre-commit-config.yaml

```diff
@@ -47,7 +45,12 @@ repos:
     rev: v0.720
     hooks:
     -   id: mypy
-        files: ^(src/|testing/)
```

Comment:
The red is preferred here (see fd8f92d0e706f4ec1d2a9815859e5e5189aaae18).

Reply:
Thanks for catching, fixed.